src/cpp/fastdds/utils/QosConverters.cpp

```diff
@@ -0,0 +1,233 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+ * QosConverters.cpp
+ *
+ */
+
```

Comment:
`#include <string>` `std::string` is being used and we try to include what we use.

Reply:
Changed in 82ac1232e Refs #15417: using std::string.